jerry-core/lit/lit-char-helpers.cpp

```diff
@@ -288,6 +288,18 @@ lit_char_is_hex_digit (ecma_char_t c) /**< code unit */
 } /* lit_char_is_hex_digit */
 
 /**
+ * Check if specified character is an ascii alpha character
+ *
+ * @return true / false
+ */
+bool
+lit_char_is_ascii_alpha (ecma_char_t c) /**< code unit */
```

Comment:
@zherczeg, seems that this function is unused now. Could you, please, remove it?